tests/js/server/aql/aql-profiler-noncluster-nightly.js

```diff
@@ -53,7 +53,7 @@ function ahuacatlProfilerTestSuite () {
     EnumerateListNode, EnumerateViewNode, FilterNode, GatherNode, IndexNode,
     InsertNode, LimitNode, NoResultsNode, RemoteNode, RemoveNode, ReplaceNode,
     ReturnNode, ScatterNode, ShortestPathNode, SingletonNode, SortNode,
-    SubqueryNode, TraversalNode, UpdateNode, UpsertNode } = profHelper;
```

Comment:
Can `SubqueryBlock` (a few lines below) be removed, too?

Reply:
Yes :+1: